src/lib.rs

```diff
@@ -94,6 +94,9 @@ pub fn generate_simple_self_signed(subject_alt_names :impl Into<Vec<String>>) ->
 // Example certs usable as reference:
 // Uses ECDSA: https://crt.sh/?asn1=607203242
 
+/// pkcs-9-at-extensionRequest in RFC 2985
+const OID_PKCS_9_AT_EXTENSION_REQUEST: &[u64] = &[1, 2, 840, 113549, 1, 9, 14];
```

Comment:
code format nit: should be `OID_PKCS_9_AT_EXTENSION_REQUEST :&[u64]`

Reply:
Fixed.